test/unit/host_test.rb

```diff
@@ -164,6 +164,20 @@ class HostTest < ActiveSupport::TestCase
     assert_equal host.vm_compute_attributes, nil
   end
 
+  test "can view hosts as non-admin user" do
+    #Testing Issue #13545
```

Comment:
I think it's better not to reference outside sources in the code, furthermore it's not clear how the issue is tested by this. Maybe explaining you want to test permissions use `Host::Managed` in the title is better?